pyam/__init__.py

```diff
@@ -10,26 +10,30 @@
 from pyam.iiasa import read_iiasa  # noqa: F401
 from pyam.datareader import read_worldbank  # noqa: F401
 
+# appease linter
+from pyam.utils import ConfigPseudoHandler
```

Comment:
don't think that this comment is relevant for future devs
```suggestion
from pyam.utils import ConfigPseudoHandler
```